tests/test_config.py

```diff
@@ -109,34 +109,25 @@ def test_default_config(self, cli_runner):  # pylint: disable=unused-argument
         self._assert_defaults(cfg)
         self._assert_model_defaults(cfg)
 
-    def test_config_missing_required_field_groups(self, tmp_path_home):
+    def test_cfg_auto_fill(self, tmp_path_home):  # pylint: disable=unused-argument
```

Comment:
are you sure the pylint rule is needed? Seems like you're using both args in the test

Reply:
sorry, will remove